integration-tests/test-backcompat.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/sh
```

Comment:
This file should be added to `.circleci/config.yml` in order to run it as part of CI, right?

Reply:
Added.